app/views/publishers/vacancies/build/schools.html.slim

```diff
@@ -14,22 +14,24 @@
 
       = render "publishers/vacancies/vacancy_form_partials/hidden_state_input", f: f
 
-      - if @multiple_schools && @school_options.count > 10
-        = f.govuk_fieldset legend: { text: t("helpers.legend.publishers_job_listing_schools_form.organisation_ids_html") } do
-          = render(Shared::SearchableCollectionCheckboxesComponent.new(form: f,
-            attribute_name: :organisation_ids,
-            collection: @school_options,
-            value_method: :id,
-            text_method: :name,
-            hint_method: :address))
-
-      - elsif @multiple_schools && @school_options.count <= 10
-        = f.govuk_collection_check_boxes :organisation_ids, @school_options, :id, :name, :address, classes: "checkbox-label__bold"
-
-      / TODO: Implement Shared::SearchableCollectionRadiobuttonsComponent
-
-      - elsif !@multiple_schools
-        = f.govuk_collection_radio_buttons :organisation_ids, @school_options, :id, :name, :address
+      = f.govuk_fieldset legend: { text: t("helpers.legend.publishers_job_listing_schools_form.organisation_ids_html") } do
```

Comment:
aren't we gonna have an extra fieldset legend when the component is not searchable?

Reply:
i was a bit confused when we paired last week why you mentioned this as when i look at the figma the fieldset legend is always there, although granted the fieldset word `school` becomes `schools` for different scenario which i need to put into this PR. https://www.figma.com/file/wyEeHQ1U9aCXOJdzh8lF8a/Sprint-72?node-id=91%3A1095 1.1.1 1.2.2